cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -110,7 +108,8 @@ public Boolean call() throws Exception {
     try {
       checkServicesWithRetry(monitorCallable, "CDAP Services are not available");
     } catch (Throwable e) {
```

Comment:
can't, right? the ``UnauthorizedException`` is the ``rootCause``, not the exception itself?